projects/simulation/human_dataset_generation/download_models.sh

```diff
@@ -0,0 +1,27 @@
+wget --load-cookies /tmp/cookies.txt "ftp://opendrdata.csd.auth.gr/simulation/human_data_generation_framework/human_models.tar.gz" -O human_models.tar.gz && rm -rf /tmp/cookies.txt
```

Comment:
```suggestion
#!/bin/bash

wget --load-cookies /tmp/cookies.txt "ftp://opendrdata.csd.auth.gr/simulation/human_data_generation_framework/human_models.tar.gz" -O human_models.tar.gz && rm -rf /tmp/cookies.txt
```

I guess we want to use bash (even though for this script this probably doesn't matter).